### PR TITLE
MINOR: Pinterest link correction

### PR DESCRIPTION
**Pull request by manjuapu**
@guozhangwang Pleas review

### CODE REVIEW

File: 0110/streams/index.html [hunk 1]
```diff
@@ -82,26 +82,8 @@ <h3 class="streams_intro">The easiest way to write mission-critical real-time ap
...
-               </a>
-               <p class="grid__item__customer__description extra__space">As the leading online fashion retailer in Europe, Zalando uses Kafka as an ESB (Enterprise Service Bus), which helps us in transitioning from a monolithic to a micro services architecture. Using Kafka for processing
-                 <a href="https://kafka-summit.org/sessions/using-kstreams-ktables-calculate-real-time-domain-rankings/" target='blank'> event streams</a> enables our technical team to do near-real time business intelligence.
-               </p>
-             </div>
-           </div>
-           <div class="customer__grid">
              <div class="customer__item streams_logo_grid streams__ny__grid">
-               <a href="https://www.nytimes.com" target="_blank" class="grid__logo__link">
+               <a href="https://www.confluent.io/blog/publishing-apache-kafka-new-york-times/" target="_blank" class="grid__logo__link">
```
Comment (guozhangwang) [inline 0110/streams/index.html:86]: Maybe better to use this url instead as it is from the original nyt website?

https://open.nytimes.com/publishing-with-apache-kafka-at-the-new-york-times-7f0e3b7d2077

File: 0110/streams/index.html [hunk 2]
```diff
@@ -111,23 +93,41 @@ <h3 class="stream__text">Streams API use cases</h3>
            </div>
            <div class="customer__grid">
              <div class="customer__item streams_logo_grid streams__ny__grid">
-               <a href="https://www.nytimes.com" target="_blank" class="grid__logo__link">
+               <a href="https://medium.com/@Pinterest_Engineering/using-kafka-streams-api-for-predictive-budgeting-9f58d206c996" target="_blank" class="grid__logo__link">
```
Comment (guozhangwang) [inline 0110/streams/index.html]: Hmm.. why we need to ref links for pinterest, with one for its logo and the content as well while for others we just use one ref link for the logo only?

**Review by guozhangwang — COMMENTED** [review]
Thanks @manjuapu , left two minor comments.

File: 0110/streams/index.html (hunk 2 above)
Comment (manjuapu) [reply 0110/streams/index.html]: @guozhangwang I have link for all logos as well as link in the content.

**Comment by manjuapu** [comment]
@guozhangwang I am working on arranging the logos on powered-by page. It is still work in progress in this PR.

**Comment by guozhangwang** [comment]
LGTM. Merged to asf-site.

**Pull request closed without merging**